src/cobalt.rs

```diff
@@ -53,11 +53,11 @@ pub fn build(config: &Config) -> Result<()> {
         .map(OsStr::new)
         .collect();
 
-    let layouts = source.join(&config.layouts);
-    let mut layouts_cache = HashMap::new();
+    let templates_dir = source.join(&config.templates);
+    let mut templates_cache_dir = HashMap::new();
```

Comment:
>  canonically, it is the "template cache directory" 

I think I'm missing something.  Which part are you referring to as a directory?

On disk, there is a "templates" directory.  We load individual templates from that and cache them in memory in this variable.  So that'd make the variable a cache of templates (`templates_cache`) or a cache of the templates directory (`templates_dir_cache`).